src/renderers/dom/shared/SVGDOMPropertyConfig.js

```diff
@@ -264,7 +264,9 @@ var ATTRS = {
   xlinkType: 'xlink:type',
   xmlBase: 'xml:base',
   xmlLang: 'xml:lang',
+  xmlNS: 'xmlns',
```

Comment:
It appears like the capitalization only happens after a `:`. Is there any reason to break that pattern here?